docker-bake.hcl

```diff
@@ -14,8 +59,17 @@ function "bindir" {
 target "_common" {
   args = {
     BUILDKIT_CONTEXT_KEEP_GIT_DIR = 1
-    APT_MIRROR = "cdn-fastly.deb.debian.org"
```

Comment:
Looks like this commit changes the default from `cdn-fastly.deb.debian.org` to `deb.debian.org`, but doesn't mention that; 

- was there a specific reason to change it? (either one should probably be fine)
- the Dockerfile itself also has handling for defaults (although that part could probably use some improvements to _not make changes_ if no custom value was set)

Would it work to "not" define a value, so that we keep the default central in the Dockerfile?

Reply:
My bad, that was not intentional. I wonder if defining an alternative mirror is still necessary though.

Reply:
We should indeed look if we still want to set this mirror. The option itself _could_ still be useful for users though to pick an alternative (thinking of people behind the GFW)